co2eq/purchase/index.js

```diff
@@ -3,6 +3,8 @@ import {
   ACTIVITY_TYPE_MEAL,
   ACTIVITY_TYPE_TRANSPORTATION,
   ACTIVITY_TYPE_PURCHASE,
+  AVERAGE_CPI_COICOPCODE_INDICATOR,
+  AVERAGE_CPI_COUNTRY_INDICATOR,
```

Comment:
those constants are only used in this file so I would declare them in this file instead of importing them